src/program/snabbvmx/doc/README.troubleshooting.md

```diff
@@ -370,38 +400,35 @@ Usage: check [-r] CONF V4-IN.PCAP V6-IN.PCAP V4-OUT.PCAP V6-OUT.PCAP
 
 Parameters:
 
-- **CONF**: SnabbVMX (icmp_snabbvmx-lwaftr-xe.cfg) or lwaftr (icmp_snabbvmx-lwaftr-xe1.conf) configuration file.
-- **V4-IN.PCAP**: Incoming IPv4 packets (from Internet).
-- **V6-IN.PCAP**: Incoming IPv6 packets (from b4).
-- **V4-OUT.PCAP**: Outgoing IPv4 packets (to Internet, decapsulated).
-- **V6-OUT.PCAP**: Outgoing IPv6 packets (to b4, encapsulated)
-- **[COUNTERS.LUA]**: Lua file with counter values. Will be regenerated via [-r] parm
-
+- **CONF** : SnabbVMX (icmp_snabbvmx-lwaftr-xe.cfg) or lwaftr (icmp_snabbvmx-lwaftr-xe1.conf) configuration file.
+- **V4-IN.PCAP** : Incoming IPv4 packets (from Internet).
+- **V6-IN.PCAP** : Incoming IPv6 packets (from b4).
+- **V4-OUT.PCAP** : Outgoing IPv4 packets (to Internet, decapsulated).
+- **V6-OUT.PCAP** : Outgoing IPv6 packets (to b4, encapsulated)
+- **[COUNTERS.LUA]** : Lua file with counter values. Will be regenerated via [-r] param
 
 ## How to run SnabbVMX interactive end-to-end test
 
-If you detected an error in the lwAFTR, the first step is to obtain the
-configuration file that SnabbVMX was using, as well as a copy of lwAFTR's
-configuration and binding table.  With that information and knowing the error
-report (ping to lwAFTR but it doesn't reply, valid softwire packet doesn't get
-decapsulated, etc), you craft a hand-made packet that meets the testing case.
+If you detected an error in the lwAFTR, the first step is to obtain the configuration file that SnabbVMX was using, as well as a copy of lwAFTR's configuration and binding table.  With that information and knowing the error report (ping to lwAFTR but it doesn't reply, valid softwire packet doesn't get decapsulated, etc), you craft a hand-made packet that meets the testing case.
```

Comment:
I don't like very long lines: they make diffs harder to read. I prefer hard-wrapped lines.